packages/grid/src/components/column/Column.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+
+export type ColumnProps = {
+  children: ReactNode;
+  widths: number[];
+  className?: string;
+  element?: keyof JSX.IntrinsicElements;
```

Comment:
do not forget put short description above each filed, it will be generate to docs